library/styleguideComponents/Message/index.js

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import styled from 'styled-components';
+import classnames from 'classnames';
+import PropTypes from 'prop-types';
+import ButtonIcon from '../../../src/components/ButtonIcon';
+import RenderIf from '../../../src/components/RenderIf';
+import Notification from '../../../src/components/Notification';
+import Close from './icons/close';
+import MessageIcon from './messageIcon';
+import { hideMessage } from '../../../src/redux/actions/message';
+import './styles.css';
+
+function Message(props) {
+    // eslint-disable-next-line no-shadow
+    const { className, style, message, variant, hideMessage } = props;
+
+    const StyledNotification = styled(Notification)`
+        position: absolute;
+        z-index: 999999;
+    `;
```

Comment:
this is a component declaration and it should be defined outside the component, this means that this component will be redefined and assigned to a new memory location each time the component rerender